ros/lib/pcp_extract_config

```diff
@@ -0,0 +1,10 @@
+disk.dev.total
+hinv.ncpu
+kernel.all.cpu.idle
+kernel.all.pressure.cpu.some.avg
+kernel.all.pressure.io.full.avg
+kernel.all.pressure.io.some.avg
+kernel.all.pressure.memory.full.avg
+kernel.all.pressure.memory.some.avg
+mem.physmem
+mem.util.available
```

Comment:
This file is very specific to a processor, is it possible to create helper directory for the processor itself and then save it there?

Reply:
That would still be under the lib directory where there are files that are being used by a specific processor like `aws_instance_types.py`. 
Once we are done with the development, we may consider keeping it under the processor directory itself along with other related files. We can skip it for now.